pyproject.toml

```
@@ -66,7 +54,7 @@ requires-python = ">=3.9"
 dependencies = [
     # TODO: update to "pin-compatible" once possible, see
     # https://github.com/FFY00/meson-python/issues/29
```

This link has become dead, we could/should update it IMO
```suggestion
    # https://github.com/mesonbuild/meson-python/issues/29
```